compositionspace/segmentation.py

```diff
@@ -30,7 +30,7 @@ def get_PCA_cumsum(self, vox_ratio_file, vox_file):
         with h5py.File(vox_file,"r") as hdf:
             group = hdf.get("Group_sm_vox_xyz_Da_spec")
             group0 = hdf.get("0")
-            spec_lst = list(list(group0.attrs.values())[2])
+            spec_lst = list(list(group0.attrs.values())[1])
```

Comment:
Is this correct, I remember that index 1 did not give me the right values, which is why I changed it.

Reply:
Actually, you might remember we got some weird results in the meeting a few days ago. This was the reason.

Reply:
Perfect, then this is good.